cli/dts/lib.deno.unstable.d.ts

```diff
@@ -167,6 +167,27 @@ declare namespace Deno {
     swapFree: number;
   }
 
+  /** **Unstable** new API. yet to be vetted.
+   *
+   * Returns the total number of logical cpus in the system along with
+   * the speed measured in MHz
+   *
+   * ```ts
+   * console.log(Deno.systemCpuInfo());
+   * ```
+   *
+   * Requires `allow-env` permission.
+   *
+   */
+  export function systemCpuInfo(): SystemCpuInfo;
+
+  export interface SystemCpuInfo {
+    /** Total number of logical cpus in the system */
+    cores: number;
+    /** The speed of the cpu measured in MHz */
+    speed: number;
```

Comment:
Seems like both of those fields are omitted when an error occurs - it should be reflected in the types and doc comment